spec/acceptance/clearance_installation_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "spec_helper"
+
+describe "Clearance Installation" do
+  around do |example|
+    current_path = File.expand_path("../", __FILE__)
+    Dir.chdir("tmp")
```

Comment:
Not sure if it matters much though, but I assume that your CWD will leak to the next run.